src/pharmpy/datainfo.py

```diff
@@ -52,6 +54,7 @@ class ColumnInfo:
         'nmtran-time',
         'nmtran-date',
     ]
+    _all_descriptors = [None, 'body weight']
```

Comment:
Should `age` also be in `_all_descriptors` since it is in the docstring?

Reply:
Yes, missed it. Good catch